lib/perl/Genome/Model/ClinSeq.pm

```diff
@@ -645,33 +645,18 @@ sub _resolve_workflow_for_build {
     }
 
     #TophatJunctionsAbsolute - Run tophat junctions absolute analysis on normal
-    if ($build->normal_rnaseq_build) {
-        my $normal_tophat_junctions_absolute_op = $self->tophat_junctions_absolute_op($workflow, 'normal');
-    }
-    #TophatJunctionsAbsolute - Run tophat junctions absolute analysis on tumor
-    my $tumor_tophat_junctions_absolute_op;
-    if ($build->tumor_rnaseq_build) {
-        $tumor_tophat_junctions_absolute_op = $self->tophat_junctions_absolute_op($workflow, 'tumor');
-    }
-
     #CufflinksExpressionAbsolute - Run cufflinks expression absolute analysis on normal
     if ($build->normal_rnaseq_build) {
+        my $normal_tophat_junctions_absolute_op = $self->tophat_junctions_absolute_op($workflow, 'normal');
```

Comment:
I presume this is why the scope of this variable is limited to the conditional block.  If the normal is defined then the downstream annotation step is linked to the differential_expression rather than these normal-only steps.  Should these operations be used elsewhere in the workflow?

I'm running a test build now with the normal RNA-seq defined.  The test @susannasiebert was using did not have a normal rnaseq input and may not have covered this case.